src/main/resources/swagger.api/records-editor.yaml

```diff
@@ -59,6 +59,26 @@ paths:
             type: string
             pattern: "[a-zA-Z]{2}"
             default: "en"
+    post:
+      responses:
+        '202':
+          description: "Request is accepted"
```

Comment:
I think that this endpoint should return `status` entity contains`qmRecordId`. I will fix it in my story if no objections.

Reply:
no objections, thanks

Reply:
well, not sure that returning status is appropriate in this case. usually POST returns the same entity that has been created or it's id